CHANGES.rst

```diff
@@ -1,6 +1,256 @@
 Traits CHANGELOG
 ================
 
+Release 6.1.0
+-------------
+
+Released: 2020-05-XX
```

Comment:
This will be updated upon actual release; there's already an entry in the release checklist to remind us to do this.